app/pulp/app/settings.py

```diff
@@ -165,18 +165,22 @@
             'simple': {'format': 'pulp: %(name)s:%(levelname)s: %(message)s'},
         },
         'handlers': {
-            'stream': {
+            'pulp': {
                 'class': 'logging.StreamHandler',
                 'formatter': 'simple',
             },
         },
         'loggers': {
-            'pulp.app': {
-                'handlers': ['stream'],
+            '': {
+                # The root logger
+                'handlers': ["pulp"],
+            },
+            'pulp': {
```

Comment:
Should we just do away with the `pulp` logger altogether? It only adds the `level=INFO` option which I think is not what we want anyway. Having a single root logger with a default `level` would probably be good.